__tests__/articles.test.js

```diff
@@ -27,7 +27,8 @@ describe("GET /api/articles/:article_id", () => {
                 body: 'I find this existence challenging',
                 created_at: '2020-07-09T20:11:00.000Z',
                 votes: 100,
-                article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+                article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+                comment_count: 11
```

Comment:
I understand why you've changed this (comment count wasn't added before you wrote this test so I imagine this test was failing? ).  Though in usual circumstances try and keep commits/changes on branches relevant to their intended task.